distribution/docs/src/main/resources/_contents/_installing/installing-intro-contents.adoc

```diff
@@ -18,9 +18,41 @@ Adding other components does not have any security/hardening implications.
 
 These are the system/environment requirements to configure _prior_ to an installation.
 
-.Hardware Requirements
-* At least 4096MB of memory for ${branding}.
-** This amount can be increased to support memory-intensive applications. See <<jvm-memory-configuration, Memory Considerations>>.
+====== Minimum and Recommended Requirements for ${branding} Systems in Production Mode
+
+.Using the Standard installation of the ${branding} application:
+[cols="2,3,3" options="header"]
+|===
+
+|*Hardware/Software Criteria*
+|*Minimum*
+|*Recommended*
+
+|Operating System
+|Solaris 10, Mac OS X 10.9 Mavericks, Windows Server 2012, Linux CentOS 5
```

Comment:
❓ Have we tested on Solaris? If not, do we really want or need to list those in the table?